ods_ci/tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot

```diff
@@ -236,28 +228,27 @@ Verify Quick Starts Work As Expected When Restarted And Left In Between
         Run Keyword And Continue On Failure     Link Text On QuickStart Card Should Be  element=${element}
         ...    exp_link_text=Restart
         Open QuickStart Element In Resource Section By Name     ${element}
-        Page Should Not Contain         //article[@id="${element}"]//span[@class="pf-v5-c-label pf-m-green pf-m-outline"]
+        Page Should Not Contain    //article[@id="${element}"]//span[@class="pf-v5-c-label pf-m-green pf-m-outline"]
         ${count}=   Get The Count Of QuickStart Steps
         Run Keyword And Continue On Failure     Click Button    //button[@data-testid="qs-drawer-start"]
         Run Keyword And Continue On Failure     Wait Until Page Contains Element
         ...    //div[@class="pfext-quick-start-content"]
-        ${temp_count}   Set Variable    2
-        FOR     ${index}    IN RANGE    ${temp_count}
-            IF  ${index} != ${count-1}
-                Run Keyword And Continue On Failure     Wait Until Keyword Succeeds    2 times   0.3s
-                ...    Mark Step Check As Yes
-                Run Keyword And Continue On Failure     Go To Next QuickStart Step
-            END
+        ${half_count}=   Set Variable    ${count // 2}
```

Comment:
isn't it missing the last step?

Reply:
No... It's just how Github is showing changes. Look at the whole file:
https://github.com/red-hat-data-services/ods-ci/blob/49cba650084009d9144c701e483143bf0b054691/ods_ci/tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot#L236-L241

Reply:
Ah, the count // 2 is the check to see what happens if we leave the quick start in the middle

Reply:
yes, previously it was set to "2" steps (hardcoded), which cant be in progress if total steps is <= 2.